neuralprophet/forecaster.py

```diff
@@ -295,6 +295,12 @@ class NeuralProphet:
             Provide `None` to deactivate the use of accelerators.
         trainer_config: dict
             Dictionary of additional trainer configuration parameters.
+        forecast_period : int
```

Comment:
`forecast_period` is often used as synonym for what we have called `n_forecasts`, so this could be ambiguous to many. 
I am having a hard time coming up with a better name. Here are some ideas:
- forecast_stride
- forecast_skip_period
- forecasting_frequency
- forecast_repeat_frequency

Reply:
@noxan Could you please give some quick feedback: which one would you pick?